gcc/rust/rust-session-manager.cc

```diff
@@ -283,7 +283,7 @@ Session::enable_dump (std::string arg)
     {
       rust_error_at (
 	Location (),
-	"dump option was not given a name. choose %<lex%>, %<parse%>, "
+	"dump option was not given a name. choose %<lex%>, "
```

Comment:
oh, I had never realized that we were parsing the string directly here instead of doing magic in `gcc/rust/lang.opt` like we do for the other enums options we've added. I think this is fine but we should eventually think about moving it here maybe? but this is not relevant to your PR so no worries

Reply:
We had another file for parsing command line args?, I will check that out. :D